gensim/test/test_doc2vec.py

```diff
@@ -76,6 +76,13 @@ def test_persistence(self):
         model.save(testfile())
         self.models_equal(model, doc2vec.Doc2Vec.load(testfile()))
 
+    def testPersistenceWord2VecFormat(self):
+        """Test storing the entire model in word2vec format."""
+        model = doc2vec.Doc2Vec(DocsLeeCorpus(), min_count=1)
+        model.save_word2vec_format(testfile(), doctag_vec=True, binary=True)
+        binary_model_dv = keyedvectors.KeyedVectors.load_word2vec_format(testfile(), binary=True)
+        self.assertEqual(len(model.wv.vocab) + len(model.docvecs), len(binary_model_dv.vocab))
```

Comment:
Please add tests for more combinations of `word_vec`/`doctag_vec` `True`/`False`